src/gui/src/mainWindow.cpp

```diff
@@ -1614,6 +1620,22 @@ void MainWindow::openDesign()
   }
 }
 
+void MainWindow::saveDesign()
+{
+  const QString filefilter = "OpenDB (*.odb *.ODB)";
+  const QString file = QFileDialog::getSaveFileName(
+      this, "Save Design", QString(), filefilter);
+
+  if (file.isEmpty()) {
+    return;
+  }
+
+  try {
+    ord::OpenRoad::openRoad()->writeDb(file.toStdString().c_str());
+  } catch (const std::exception&) {
+  }
```

Comment:
On an exception we should give some indication - perhaps an error dialog with the message.

Reply:
if an error occurs it will be printed via the logger, does it make sense to also display a dialog box? we don't do that anywhere else in the gui code.
The reason for the try/catch is to avoid a fault when openroad throws an exception during an error.

Reply:
Ok so long as it appears somewhere